lib/public/views/Error/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { h, iconHome } from '/js/src/index.js';
+
+/**
+ * Error page component that dynamically displays error details based on the model
+ * @param {Model} model - Represents the current application state
+ * @returns {Component} Error page component
+ */
+export const ErrorPage = (model) => {
+    const { code, codeDescription, message } = model.errorModel.error;
+    return h('div.flex-column.justify-center ', [
+        h('.flex-column.items-center.g3.mv4', [
+            h('img', {
+                src: 'assets/alice.png',
+                alt: 'Alice logo',
+                style: 'width: 200px',
+            }),
+            h('h2', 'Oops! Something went wrong.'),
+            h('h3', `${code} - ${codeDescription}`),
+            h('.f5', message),
+            h('button.btn.btn-primary', {
+                onclick: () => {
+                    model.router.go('?page=home');
```

Comment:
In my opinion, we should avoid as much as possible dependencies inside components. The ideal would be for it to depend only on a very specific submodel, in this case only on `model.errorModel`. Using router is an external dependency that should be avoided outside of models.
Instead, you should try to have a look at "frontLink" component, that hides this dependency for you (and puts it in one single place). You might style it as a button if you like, but in my personal opinion, a link should look like a link (I am fine with the button if you prefer it).